interview-questions.md

```diff
@@ -6,52 +6,62 @@ First, without external resources. Challenge yourself to answer from memory as i
 
 Then, research the question to expand on your answer. Even if you feel you have answered the question completely, there is always something more to learn. Write your researched answer in your OWN WORDS.
 
-INSTRUCTOR EXAMPLE: What is a conditional statement?
-
-Your answer:
-
-Researched answer:
 
 1. What is git? What is the difference between git and Github?
 
 Your answer:
+Git is the version of code that we use to create/edit.  GitHub is the database where Git is submitted and used to store, modify, and or merge various versions of code. GitHub can allow multiple developers to obtain code to work on that has been submitted.
 
 Researched answer:
+Git is a version control system that we use to manage creation/modification of source code.  GitHub is a cloud-based hosting service that makes it easier to manage Git repositories.  This makes collaboration easier and more organized.
 
 2. Which JavaScript operators will return a Boolean value?
 
 Your answer:
+Any operator that returns an value that is either true or false. For example, my name is John.  If I'm asked, and I reply with 'John', the Boolean return is true.  An answer that isn't 'John', would be false.
 
 Researched answer:
+The Boolean value of an expression is the basis for all Javascript comparisons and conditions that can only have one of two values.
```

Comment:
You should expand on this a bit more and use some tech vocab. You are right in that they can be used for comparisons, but what are they?